apps/builder/app/shared/shortcuts/shortcuts.ts

```diff
@@ -37,8 +37,11 @@ export const useSharedShortcuts = () => {
   useHotkeys(
     "backspace, delete",
     deleteSelectedInstance,
-    // prevent instance deletion while deleting text
-    { enableOnFormTags: false, enableOnContentEditable: false },
+    {
+      // prevent instance deletion while deleting text in a control at style panel etc.
```

Comment:
"and allow deleting of input component on canvas while get focused when selected"

Reply:
Added